canonical/charmed-kubeflow.io
# chore: Update canonicalwebteam.discourse to 5.5.0

## Done

- Update canonicalwebteam.discourse to 5.5.0

## QA

- Go to https://charmed-kubeflow-io-178.demos.haus/docs
- Inspect a heading (h2 and down) and see the anchor link within it doesn't have trailing number
